packages/web/src/scss/theme/_form-fields.scss

```diff
@@ -27,7 +28,8 @@ $box-field-input-border-color-default: tokens.$border-primary-default;
 $box-field-input-border-color-focus: tokens.$focus-default;
 $box-field-input-border-color-success: tokens.$emotion-success-default;
 $box-field-input-border-color-warning: tokens.$emotion-warning-default;
-$box-field-input-border-color-error: tokens.$emotion-danger-default;
+$box-field-input-border-color-error: tokens.$emotion-danger-default; // Deprecated (#DS-590)
```

Comment:
Please keep the deprecation comment format the same

Reply:
forgot, fixed